src/components/Modals/Onboarding/Onboarding.scss

```diff
@@ -389,10 +389,10 @@ input {
   }
   &-select {
     background: $white;
-    border: 1px solid $inactiveGray;
+    border: 0.5px solid $inactiveGray;
```

Comment:
Like the rest of these fixes, thanks for paying such close attention here! I'm glad we're getting on top of these UI changes so soon